Sofia-Rojas/tarea5/puzzle.js

```diff
@@ -0,0 +1,94 @@
+
+
+tile=[[15,8,1,4],[9,2,14,3],[12,7,5,11],[6,10,13,0]];
+tileSolution=[[1,2,3,4],[5,6,7,8],[9,10,11,12],[13,14,15,0]];
+
+function puzzle(row,column){  
+   let sideReview = slide(row,column);
+   for  (let i=0; i<sideReview.length;i++){
```

Comment:
@sofia-rojas , use modern approach like forEach, map or reduce, also remember proper name for i variable